### PR TITLE
[GPU] Added blocked layouts support for roi_pooling operations 

### PR DESCRIPTION
**Pull request by mdovbnia-lohika**
### Details:
[GPU] Added blocked layouts support for roi_pooling operations

### CODE REVIEW

**Review by mhnap-lohika — APPROVED** [review]

**Review by tgubanova-lohika — COMMENTED** [review]
could you add primitive id to program.cpp

**Comment by wilson-seok** [comment]
@mdovbnia-lohika Build error happens in build_rhel82_release and build_ubuntu18_release. Please check it.
image

**Comment by mdovbnia-lohika** [comment]
please, take a look to this new fixed version

**Comment by wilson-seok** [comment]
No performance regression in int8 models

**Review by wilson-seok — APPROVED** [review]

File: src/plugins/intel_gpu/src/graph/program.cpp [hunk 1]
```diff
@@ -1445,7 +1445,9 @@ void program::set_layout_optimizer_attributes(layout_optimizer& lo) {
             prim.type() != cldnn::adaptive_pooling::type_id() &&
             prim.type() != cldnn::bucketize::type_id() &&
             prim.type() != cldnn::roll::type_id() &&
+            prim.type() != cldnn::resample::type_id() &&
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/program.cpp:1448]: why resample is changed too?

Comment (mdovbnia-lohika) [reply src/plugins/intel_gpu/src/graph/program.cpp]: fixed

**Pull request merged**